controller-server/src/main/java/com/yahoo/vespa/hosted/controller/deployment/JobMetrics.java

```diff
@@ -0,0 +1,64 @@
+package com.yahoo.vespa.hosted.controller.deployment;
```

Comment:
Missing copyright header:

`// Copyright 2020 Oath Inc. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.`

Reply:
```suggestion
// Copyright 2020 Oath Inc. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
package com.yahoo.vespa.hosted.controller.deployment;
```